setup.py

```diff
@@ -576,31 +585,29 @@ def pxd(name):
                      '_libs/tslibs/conversion']},
     '_libs.tslibs.parsing': {
         'pyxfile': '_libs/tslibs/parsing',
-        'pxdfiles': ['_libs/src/util',
```

Comment:
pls don’t change this;  if u look in the source these depend on these

Reply:
Are we looking at different source files?  tslibs.parsing doesn't cimport from util or khash.

Reply:
you are right here